### PR TITLE
🗡️ Accept SNPE DLC model for qnn context binary generator

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes

Accept SNPE DLC model for qnn context binnary generator

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.
- [ ] Is this PR including examples changes? If yes, please remember to update [example documentation](https://github.com/microsoft/Olive/blob/main/docs/source/examples.md) in a follow-up PR.

### CODE REVIEW

File: test/unit_test/passes/qnn/test_qnn_context_bin_generator.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        {
+            "backend": "libQnnHtp.so",
+        },
+        {
+            "backend": "libQnnHtp.so",
+            "extra_args": "--dlc_path extra_test.dlc",
+        },
+    ],
+)
+def test_snpe_model_input(mocked_qnn_sdk_runner, config, tmp_path):
```
Comment (jambayk) [inline test/unit_test/passes/qnn/test_qnn_context_bin_generator.py:24]: looks like this is not supported on Windows?

Comment (trajepl) [reply test/unit_test/passes/qnn/test_qnn_context_bin_generator.py:24]: Oh, true. I skip it in my local commit but forget to push. 🤣

Comment (trajepl) [reply test/unit_test/passes/qnn/test_qnn_context_bin_generator.py:24]: image

**Review by jambayk — APPROVED** [review]

**Comment by trajepl** [comment]
Do not delete this branch until new version is release.

**Pull request merged**